addons/mail/static/tests/chatter/web/follower_list_menu.test.js

```diff
@@ -236,20 +236,20 @@ test("Load 100 recipients at once", async () => {
     );
     await start();
     await openFormView("res.partner", partnerIds[0]);
-    await contains("button[title='Show Followers']", { text: "210" });
+    await contains("button[title='Show Followers']", { text: "60" });
     await click("button", { text: "Send message" });
     await contains(".o-mail-Chatter", {
-        text: "To: partner1, partner2, partner3, partner4, partner5, and 95 more",
+        text: "To: partner1, partner2, partner10, partner11, partner12, and 15 more",
```

Comment:
Shouldn't this be `To: partner1, partner10, partner11, partner12, partner2, and 15 more`?